modules/app-gateway-module/main.tf

```diff
@@ -103,7 +104,8 @@ resource "azurerm_application_gateway" "ag" {
 
   dynamic "request_routing_rule" {
     for_each = [for app in var.frontends : {
-      name = app.name
+      name                  = app.name
```

Comment:
Can’t we just add it for everything? We don’t want special cases in here.

Reply:
If that's ok @timja. I thought it would be preferred this way.

Reply:
this functionality is generic enough that it looks fine to add for everyone